esphome/components/lilygo_t5_47_display/LilygoT547Display.h

```diff
@@ -44,6 +44,8 @@ class LilygoT547Display : public PollingComponent, public display::DisplayBuffer
   void poweron();
   void poweroff();
   void on_shutdown() override;
+  
+  display::DisplayType get_display_type() override { return display::DisplayType::DISPLAY_TYPE_BINARY; }
```

Comment:
I don't know whether your implementation supports grayscale or only binary display; if grayscale is supported, you should use that instead:
```suggestion
  display::DisplayType get_display_type() override { return display::DisplayType::DISPLAY_TYPE_GRAYSCALE; }
```

Reply:
Thanks for a great contribution! 
Implementation supports grayscale, so let's go ahead with it :)